services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -107,10 +101,10 @@ public BeaconChainController(
     this.config = config;
     this.metricsSystem = metricsSystem;
     this.testMode = config.getDepositMode().equals(DEPOSIT_TEST);
-    this.eventBus.register(this);
   }
 
   public void initAll() {
+    this.eventBus.register(this);
```

Comment:
Yay! Love seeing these move out of constructors.  Could easily be the source of your failing AT though as it will now miss events from the storage service at startup.